.github/workflows/ui-build.yml

```diff
@@ -39,3 +39,7 @@ jobs:
       - name: Build OSS app
         working-directory: ./src/ui/app
         run: npm run build
+
+      - name: Build Storybook
+        working-directory: ./src/ui/common
+        run: npm run build-storybook
```

Comment:
awesome!

Reply:
hopefully we can catch these issues when they come up in the future.